apps/mobile/src/screens/SettingsScreen/NotificationSettingsScreen.tsx

```diff
@@ -85,3 +326,22 @@ export function NotificationSettingsScreen() {
     </View>
   );
 }
+
+const DISABLED_TOGGLE_BY_EMAIL_STATUS = ['Unverified', 'Failed'];
+
+type EmailSettings = {
+  readonly unsubscribedFromAll: boolean;
+  readonly unsubscribedFromDigest: boolean;
+  readonly unsubscribedFromMarketing: boolean;
+  readonly unsubscribedFromMembersClub: boolean;
+  readonly unsubscribedFromNotifications: boolean;
+};
+
+type EmailNotificationSettings = {
+  notifications: boolean;
+  marketing: boolean;
+  digest: boolean;
+  membersClub: boolean;
+  all: boolean;
+  [key: string]: boolean;
```

Comment:
do we need this line?

Reply:
yeah adding the index signature just fixes this ts flag 
<img width="1125" alt="Screenshot 2024-03-12 at 15 19 15" src="https://github.com/gallery-so/gallery/assets/49758803/cbc2c203-4ed3-4dd4-b77d-60cfbc306e31">